### PR TITLE
Add seriesCard

### PR DESCRIPTION
**Pull request by infoucarsinan**
Create seriesCard

### CODE REVIEW

File: src/app/components/SeriesCard/SeriesCard.module.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.container {
+  background: transparent;
```
Comment (lmachens) [inline src/app/components/SeriesCard/SeriesCard.module.css]: I don't think this is required

File: src/app/components/SeriesCard/SeriesCard.module.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.seriesInfo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 0.5em;
+}
+.headline {
+  grid-column: 2/3;
```
Comment (lmachens) [inline src/app/components/SeriesCard/SeriesCard.module.css]: Are you sure you need this?

File: src/app/components/SeriesCard/SeriesCard.module.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  font-weight: 400;
+  margin: 0;
+}
+
+.seriesEpisode {
+  grid-column: 2/3;
+  color: var(--hint-color);
+  font-weight: 400;
```
Comment (lmachens) [inline src/app/components/SeriesCard/SeriesCard.module.css]: I guess this is the default font-weight -> You can remove it.

File: src/app/components/SeriesCard/SeriesCard.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+type InputProps = {
+  headline: string;
+  seriesInfo: string;
+  imgSrc: string;
+};
+
+function SeriesCard({ headline, seriesInfo, imgSrc }: InputProps): JSX.Element {
+  return (
+    <div className={styles.container}>
```
Comment (lmachens) [inline src/app/components/SeriesCard/SeriesCard.tsx]: I like to see a semantic tag here like `article`.

File: src/app/components/SeriesCard/SeriesCard.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+      <img className={styles.img} src={imgSrc} />
```
Comment (lmachens) [inline src/app/components/SeriesCard/SeriesCard.tsx]: Don't forget about `alt`.

File: src/app/components/SeriesCard/SeriesCard.module.css [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  grid-column: 1/2;
+}
+
+.seriesInfo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 0.5em;
+}
+.headline {
```
Comment (lmachens) [inline src/app/components/SeriesCard/SeriesCard.module.css]: This is not exactly BEM like. 
Maybe this is better?
```suggestion
.seriesInfo__headline {
```

**Review by lmachens — CHANGES_REQUESTED** [review]
Good job (oﾟvﾟ)ノ. Can you link the related issue in the pull request description?

**Review by SGierszewski — APPROVED** [review]
Good job!! 🥳

**Review by MaxMuecket — APPROVED** [review]
well done👌

**Comment by J-Bannach** [comment]
👍

File: src/app/components/SeriesCard/SeriesCard.module.css (hunk 1 above)
Comment (infoucarsinan) [reply src/app/components/SeriesCard/SeriesCard.module.css]: ok

File: src/app/components/SeriesCard/SeriesCard.module.css (hunk 6 above)
Comment (infoucarsinan) [reply src/app/components/SeriesCard/SeriesCard.module.css]: ok

File: src/app/components/SeriesCard/SeriesCard.module.css (hunk 3 above)
Comment (infoucarsinan) [reply src/app/components/SeriesCard/SeriesCard.module.css]: ok

File: src/app/components/SeriesCard/SeriesCard.tsx (hunk 4 above)
Comment (infoucarsinan) [reply src/app/components/SeriesCard/SeriesCard.tsx]: ok

File: src/app/components/SeriesCard/SeriesCard.module.css (hunk 2 above)
Comment (infoucarsinan) [reply src/app/components/SeriesCard/SeriesCard.module.css]: ok

**Pull request merged**